linux_os/guide/system/network/network-ipv6/disabling_ipv6/sysctl_net_ipv6_conf_all_disable_ipv6/rule.yml

```diff
@@ -20,7 +20,8 @@ identifiers:
     cce@rhel7: CCE-80175-3
 
 references:
-    cis: 3.3.3
+    cis@rhel7: 3.3.3
+    cis@rhel8: 3.3.3
```

Comment:
```suggestion
```

We go with the approach of disabling the kernel module, instead of setting the sysctl.
As far as I remember this approach was to avoid issues....